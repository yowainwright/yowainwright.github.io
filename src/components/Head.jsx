import React from 'react'
import Helmet from 'react-helmet'

export const Header = ({
  gtmInlineScript = "window.dataLayer = window.dataLayer || [];function gtag(){dataLayer.push(arguments);}gtag('js', new Date());gtag('config', 'UA-73077309-1');",
}) => (
  <Helmet>
    <meta name='robots' content='index,follow' />
    <meta property='og:locale' content='en_US' />
    <link href='https://jeffry.in/assets/favicon.png' rel='icon' />
    <link href='https://yowainwright.imgix.net/apple-icon-120x120.png' rel='apple-touch-icon' itemProp='logo' />
    <link rel='preconnect' href='https://fonts.googleapis.com' />
    <link rel='preconnect' href='https://fonts.gstatic.com' crossOrigin />
    <link href='https://fonts.googleapis.com/css2?family=Roboto:wght@400;700&display=swap' rel='stylesheet' />
    <meta
      name='keywords'
      content='jeffry, wainwright, jeffry wainwright, code, programmer, artist, athlete, developer, engineer'
    />
    <meta name='theme-color' content='#ffffcc' />
    <link rel='manifest' href='https://yowainwright.imgix.net/manifest.json' />
    <meta name='twitter:card' content='summary_large_image' />
    <meta name='twitter:site' content='@yowainwright' />
    <meta name='twitter:creator' content='@yowainwright' />
    <meta property='og:site_name' content='Jeffry.in' />
    <meta property='og:image:width' content='1200' />
    <meta property='og:image:height' content='600' />
    <meta name='google-site-verification' content='cXTq9c3NhBvHJsPXxzWAYAqbB8PRUKUxemU8mykg_vs' />

    <script src='https://www.googletagmanager.com/gtag/js?id=UA-73077309-1' />
    <script>{gtmInlineScript}</script>
  </Helmet>
)

export default Header
