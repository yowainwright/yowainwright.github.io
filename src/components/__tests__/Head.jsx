import React from 'react'
import renderer from 'react-test-renderer'
import Head from '../Head'

describe('Head component', () => {
  it('renders correctly', () => {
    const rendered = renderer.create(<Head />).toJSON()

    expect(rendered).toMatchSnapshot()
  })
})
