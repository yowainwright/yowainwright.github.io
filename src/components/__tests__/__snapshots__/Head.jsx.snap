// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Head component renders correctly 1`] = `
ShallowWrapper {
  "length": 1,
  Symbol(enzyme.__root__): [Circular],
  Symbol(enzyme.__unrendered__): <Head />,
  Symbol(enzyme.__renderer__): Object {
    "batchedUpdates": [Function],
    "getNode": [Function],
    "render": [Function],
    "simulateEvent": [Function],
    "unmount": [Function],
  },
  Symbol(enzyme.__node__): Object {
    "instance": null,
    "key": undefined,
    "nodeType": "class",
    "props": Object {
      "children": Array [
        <meta
          content="index,follow"
          name="robots"
        />,
        <meta
          content="en_US"
          property="og:locale"
        />,
        <link
          href="https://jeffry.in/assets/favicon.ico"
          rel="icon"
        />,
        <link
          href="https://yowainwright.imgix.net/w.png"
          itemProp="logo"
          rel="apple-touch-icon"
        />,
        <meta
          content="jeffry, wainwright, jeffry wainwright, code, programmer, artist, athlete, developer, engineer"
          name="keywords"
        />,
        <meta
          content="#ffffcc"
          name="theme-color"
        />,
        <link
          href="https://yowainwright.imgix.net/manifest.json"
          rel="manifest"
        />,
        <meta
          content="summary_large_image"
          name="twitter:card"
        />,
        <meta
          content="@yowainwright"
          name="twitter:site"
        />,
        <meta
          content="@yowainwright"
          name="twitter:creator"
        />,
        <meta
          content="Jeffry.in"
          property="og:site_name"
        />,
        <meta
          content="1200"
          property="og:image:width"
        />,
        <meta
          content="600"
          property="og:image:height"
        />,
        <meta
          content="cXTq9c3NhBvHJsPXxzWAYAqbB8PRUKUxemU8mykg_vs"
          name="google-site-verification"
        />,
        <script
          src="https://www.googletagmanager.com/gtag/js?id=UA-73077309-1"
        />,
        <script>
          window.dataLayer = window.dataLayer || [];function gtag(){dataLayer.push(arguments);}gtag('js', new Date());gtag('config', 'UA-73077309-1');
        </script>,
      ],
      "defer": true,
      "encodeSpecialCharacters": true,
    },
    "ref": null,
    "rendered": Array [
      Object {
        "instance": null,
        "key": undefined,
        "nodeType": "host",
        "props": Object {
          "content": "index,follow",
          "name": "robots",
        },
        "ref": null,
        "rendered": null,
        "type": "meta",
      },
      Object {
        "instance": null,
        "key": undefined,
        "nodeType": "host",
        "props": Object {
          "content": "en_US",
          "property": "og:locale",
        },
        "ref": null,
        "rendered": null,
        "type": "meta",
      },
      Object {
        "instance": null,
        "key": undefined,
        "nodeType": "host",
        "props": Object {
          "href": "https://jeffry.in/assets/favicon.ico",
          "rel": "icon",
        },
        "ref": null,
        "rendered": null,
        "type": "link",
      },
      Object {
        "instance": null,
        "key": undefined,
        "nodeType": "host",
        "props": Object {
          "href": "https://yowainwright.imgix.net/w.png",
          "itemProp": "logo",
          "rel": "apple-touch-icon",
        },
        "ref": null,
        "rendered": null,
        "type": "link",
      },
      Object {
        "instance": null,
        "key": undefined,
        "nodeType": "host",
        "props": Object {
          "content": "jeffry, wainwright, jeffry wainwright, code, programmer, artist, athlete, developer, engineer",
          "name": "keywords",
        },
        "ref": null,
        "rendered": null,
        "type": "meta",
      },
      Object {
        "instance": null,
        "key": undefined,
        "nodeType": "host",
        "props": Object {
          "content": "#ffffcc",
          "name": "theme-color",
        },
        "ref": null,
        "rendered": null,
        "type": "meta",
      },
      Object {
        "instance": null,
        "key": undefined,
        "nodeType": "host",
        "props": Object {
          "href": "https://yowainwright.imgix.net/manifest.json",
          "rel": "manifest",
        },
        "ref": null,
        "rendered": null,
        "type": "link",
      },
      Object {
        "instance": null,
        "key": undefined,
        "nodeType": "host",
        "props": Object {
          "content": "summary_large_image",
          "name": "twitter:card",
        },
        "ref": null,
        "rendered": null,
        "type": "meta",
      },
      Object {
        "instance": null,
        "key": undefined,
        "nodeType": "host",
        "props": Object {
          "content": "@yowainwright",
          "name": "twitter:site",
        },
        "ref": null,
        "rendered": null,
        "type": "meta",
      },
      Object {
        "instance": null,
        "key": undefined,
        "nodeType": "host",
        "props": Object {
          "content": "@yowainwright",
          "name": "twitter:creator",
        },
        "ref": null,
        "rendered": null,
        "type": "meta",
      },
      Object {
        "instance": null,
        "key": undefined,
        "nodeType": "host",
        "props": Object {
          "content": "Jeffry.in",
          "property": "og:site_name",
        },
        "ref": null,
        "rendered": null,
        "type": "meta",
      },
      Object {
        "instance": null,
        "key": undefined,
        "nodeType": "host",
        "props": Object {
          "content": "1200",
          "property": "og:image:width",
        },
        "ref": null,
        "rendered": null,
        "type": "meta",
      },
      Object {
        "instance": null,
        "key": undefined,
        "nodeType": "host",
        "props": Object {
          "content": "600",
          "property": "og:image:height",
        },
        "ref": null,
        "rendered": null,
        "type": "meta",
      },
      Object {
        "instance": null,
        "key": undefined,
        "nodeType": "host",
        "props": Object {
          "content": "cXTq9c3NhBvHJsPXxzWAYAqbB8PRUKUxemU8mykg_vs",
          "name": "google-site-verification",
        },
        "ref": null,
        "rendered": null,
        "type": "meta",
      },
      Object {
        "instance": null,
        "key": undefined,
        "nodeType": "host",
        "props": Object {
          "src": "https://www.googletagmanager.com/gtag/js?id=UA-73077309-1",
        },
        "ref": null,
        "rendered": null,
        "type": "script",
      },
      Object {
        "instance": null,
        "key": undefined,
        "nodeType": "host",
        "props": Object {
          "children": "window.dataLayer = window.dataLayer || [];function gtag(){dataLayer.push(arguments);}gtag('js', new Date());gtag('config', 'UA-73077309-1');",
        },
        "ref": null,
        "rendered": "window.dataLayer = window.dataLayer || [];function gtag(){dataLayer.push(arguments);}gtag('js', new Date());gtag('config', 'UA-73077309-1');",
        "type": "script",
      },
    ],
    "type": [Function],
  },
  Symbol(enzyme.__nodes__): Array [
    Object {
      "instance": null,
      "key": undefined,
      "nodeType": "class",
      "props": Object {
        "children": Array [
          <meta
            content="index,follow"
            name="robots"
          />,
          <meta
            content="en_US"
            property="og:locale"
          />,
          <link
            href="https://jeffry.in/assets/favicon.ico"
            rel="icon"
          />,
          <link
            href="https://yowainwright.imgix.net/w.png"
            itemProp="logo"
            rel="apple-touch-icon"
          />,
          <meta
            content="jeffry, wainwright, jeffry wainwright, code, programmer, artist, athlete, developer, engineer"
            name="keywords"
          />,
          <meta
            content="#ffffcc"
            name="theme-color"
          />,
          <link
            href="https://yowainwright.imgix.net/manifest.json"
            rel="manifest"
          />,
          <meta
            content="summary_large_image"
            name="twitter:card"
          />,
          <meta
            content="@yowainwright"
            name="twitter:site"
          />,
          <meta
            content="@yowainwright"
            name="twitter:creator"
          />,
          <meta
            content="Jeffry.in"
            property="og:site_name"
          />,
          <meta
            content="1200"
            property="og:image:width"
          />,
          <meta
            content="600"
            property="og:image:height"
          />,
          <meta
            content="cXTq9c3NhBvHJsPXxzWAYAqbB8PRUKUxemU8mykg_vs"
            name="google-site-verification"
          />,
          <script
            src="https://www.googletagmanager.com/gtag/js?id=UA-73077309-1"
          />,
          <script>
            window.dataLayer = window.dataLayer || [];function gtag(){dataLayer.push(arguments);}gtag('js', new Date());gtag('config', 'UA-73077309-1');
          </script>,
        ],
        "defer": true,
        "encodeSpecialCharacters": true,
      },
      "ref": null,
      "rendered": Array [
        Object {
          "instance": null,
          "key": undefined,
          "nodeType": "host",
          "props": Object {
            "content": "index,follow",
            "name": "robots",
          },
          "ref": null,
          "rendered": null,
          "type": "meta",
        },
        Object {
          "instance": null,
          "key": undefined,
          "nodeType": "host",
          "props": Object {
            "content": "en_US",
            "property": "og:locale",
          },
          "ref": null,
          "rendered": null,
          "type": "meta",
        },
        Object {
          "instance": null,
          "key": undefined,
          "nodeType": "host",
          "props": Object {
            "href": "https://jeffry.in/assets/favicon.ico",
            "rel": "icon",
          },
          "ref": null,
          "rendered": null,
          "type": "link",
        },
        Object {
          "instance": null,
          "key": undefined,
          "nodeType": "host",
          "props": Object {
            "href": "https://yowainwright.imgix.net/w.png",
            "itemProp": "logo",
            "rel": "apple-touch-icon",
          },
          "ref": null,
          "rendered": null,
          "type": "link",
        },
        Object {
          "instance": null,
          "key": undefined,
          "nodeType": "host",
          "props": Object {
            "content": "jeffry, wainwright, jeffry wainwright, code, programmer, artist, athlete, developer, engineer",
            "name": "keywords",
          },
          "ref": null,
          "rendered": null,
          "type": "meta",
        },
        Object {
          "instance": null,
          "key": undefined,
          "nodeType": "host",
          "props": Object {
            "content": "#ffffcc",
            "name": "theme-color",
          },
          "ref": null,
          "rendered": null,
          "type": "meta",
        },
        Object {
          "instance": null,
          "key": undefined,
          "nodeType": "host",
          "props": Object {
            "href": "https://yowainwright.imgix.net/manifest.json",
            "rel": "manifest",
          },
          "ref": null,
          "rendered": null,
          "type": "link",
        },
        Object {
          "instance": null,
          "key": undefined,
          "nodeType": "host",
          "props": Object {
            "content": "summary_large_image",
            "name": "twitter:card",
          },
          "ref": null,
          "rendered": null,
          "type": "meta",
        },
        Object {
          "instance": null,
          "key": undefined,
          "nodeType": "host",
          "props": Object {
            "content": "@yowainwright",
            "name": "twitter:site",
          },
          "ref": null,
          "rendered": null,
          "type": "meta",
        },
        Object {
          "instance": null,
          "key": undefined,
          "nodeType": "host",
          "props": Object {
            "content": "@yowainwright",
            "name": "twitter:creator",
          },
          "ref": null,
          "rendered": null,
          "type": "meta",
        },
        Object {
          "instance": null,
          "key": undefined,
          "nodeType": "host",
          "props": Object {
            "content": "Jeffry.in",
            "property": "og:site_name",
          },
          "ref": null,
          "rendered": null,
          "type": "meta",
        },
        Object {
          "instance": null,
          "key": undefined,
          "nodeType": "host",
          "props": Object {
            "content": "1200",
            "property": "og:image:width",
          },
          "ref": null,
          "rendered": null,
          "type": "meta",
        },
        Object {
          "instance": null,
          "key": undefined,
          "nodeType": "host",
          "props": Object {
            "content": "600",
            "property": "og:image:height",
          },
          "ref": null,
          "rendered": null,
          "type": "meta",
        },
        Object {
          "instance": null,
          "key": undefined,
          "nodeType": "host",
          "props": Object {
            "content": "cXTq9c3NhBvHJsPXxzWAYAqbB8PRUKUxemU8mykg_vs",
            "name": "google-site-verification",
          },
          "ref": null,
          "rendered": null,
          "type": "meta",
        },
        Object {
          "instance": null,
          "key": undefined,
          "nodeType": "host",
          "props": Object {
            "src": "https://www.googletagmanager.com/gtag/js?id=UA-73077309-1",
          },
          "ref": null,
          "rendered": null,
          "type": "script",
        },
        Object {
          "instance": null,
          "key": undefined,
          "nodeType": "host",
          "props": Object {
            "children": "window.dataLayer = window.dataLayer || [];function gtag(){dataLayer.push(arguments);}gtag('js', new Date());gtag('config', 'UA-73077309-1');",
          },
          "ref": null,
          "rendered": "window.dataLayer = window.dataLayer || [];function gtag(){dataLayer.push(arguments);}gtag('js', new Date());gtag('config', 'UA-73077309-1');",
          "type": "script",
        },
      ],
      "type": [Function],
    },
  ],
  Symbol(enzyme.__options__): Object {
    "adapter": ReactSixteenAdapter {
      "options": Object {
        "enableComponentDidUpdateOnSetState": true,
      },
    },
  },
}
`;
